pkg/sqlite/stranded_test.go

```diff
@@ -0,0 +1,76 @@
+package sqlite
+
+import (
+	"context"
+	"database/sql"
+	"testing"
+
+	"github.com/operator-framework/operator-registry/pkg/image"
+	"github.com/operator-framework/operator-registry/pkg/registry"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStrandedBundleRemover(t *testing.T) {
```

Comment:
I assume because of FOREIGN KEY with ON DELETE CASCADE, all dependencies + properties associated with the stranded operators should be removed. However, it would be nice if this test case can verify that as well.